pygitguardian/models.py

```diff
@@ -286,6 +288,8 @@ def __init__(
         matches: List[Match],
         known_secret: bool = False,
         incident_url: Optional[str] = None,
+        is_excluded: Optional[bool] = False,
```

Comment:
Since `is_excluded` default value is False, maybe it does not have to be `Optional`?

Reply:
Right, fixed.